homeassistant/helpers/service.py

```diff
@@ -995,10 +995,10 @@ def async_register_admin_service(
     )
 
 
-@bind_hass
+@deprecated_hass_binding(breaks_in_ha_version="2026.2")
```

Comment:
We need a year of deprecation since users can't fix this themselves, they need custom integration authors to update their integrations.
```suggestion
@deprecated_hass_binding(breaks_in_ha_version="2026.10")
```